dsgrid/config/project_config.py

```diff
@@ -158,12 +165,11 @@ def check_model_sector(cls, model_sector, values):
 class DimensionMappingsModel(DSGBaseModel):
     """Defines all dimension mappings associated with a dsgrid project, including base-to-base, base-to-supplemental, and dataset-to-project mappings."""
 
-    base_to_base: Optional[List[DimensionMappingReferenceModel]] = Field(
+    base_to_base: List[DimensionMappingReferenceModel] = Field(
```

Comment:
@mooneyme Is this field now defunct?

Reply:
this is now what we are calling an association